presidio-analyzer/tests/test_ip_recognizer.py

```diff
@@ -14,11 +15,7 @@ def test_valid_ipv4(self):
         results = ip_recognizer.analyze(context + ip, entities)
 
         assert len(results) == 1
-        assert 0.59 < results[0].score < 0.8
-        assert results[0].entity_type == entities[0]
-        assert results[0].start == 14
-        assert results[0].end == 25
-
+        assert_result_within_score_range(results[0], entities[0], 14, 25, 0.6, 0.81)
```

Comment:
Why shouldn't a test return a deterministic result? In other words, why do we need a score range?

Reply:
2 reasons:
1. you want to allow some flexibility when adding context similarity 
2. Python precision sucks, so when adding similarity text for example, you may end up expecting score 0.6 but actually it is 0.6000001